oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStore.java

```diff
@@ -642,7 +642,7 @@ public DocumentNodeStore(DocumentNodeStoreBuilder<?> builder) {
         this.branches = new UnmergedBranches();
         this.asyncDelay = builder.getAsyncDelay();
         this.versionGarbageCollector = new VersionGarbageCollector(
-                this, builder.createVersionGCSupport(), isDetailedGCEnabled(builder), false);
+                this, builder.createVersionGCSupport(), isDetailedGCEnabled(builder), false, false);
```

Comment:
Is there a particular reason for switching the default to `false`? I would have preferred to keep it at `true`.

Reply:
I agree that `true` makes more sense here and would use that value while making code changes for https://issues.apache.org/jira/browse/OAK-10633 ticket.